packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1728,6 +1739,130 @@ describe("SharedTree", () => {
 			assert.equal(trees[0].checkout.forest instanceof ChunkedForest, true);
 		});
 	});
+	describe("Schema based op encoding", () => {
+		it("uses the correct schema for subsequent edits after schema change.", async () => {
+			const factory = new SharedTreeFactory({
+				jsonValidator: typeboxValidator,
+				treeEncodeType: TreeCompressionStrategy.Compressed,
+			});
+			const provider = new TestTreeProviderLite(1, factory);
```

Comment:
I'd make there by > 1 tree to test remote ops in this case

Reply:
Updated test to more than one client